### PR TITLE
Added YAML parser node

### PR DESCRIPTION
**Pull request by natcl**

### CODE REVIEW

**Comment by natcl** [comment]
Note: put my name as copyright but I assume it should be the foundation instead ? Feel free to change !

File: nodes/core/parsers/70-YAML.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<!--
```
Comment (dceejay) [inline nodes/core/parsers/70-YAML.html]: No need for copyright section anymore :-)... we can make do with one at the top level

File: nodes/core/parsers/70-YAML.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/**
```
Comment (dceejay) [inline nodes/core/parsers/70-YAML.js]: Likewise - no need for copyright in here either

File: nodes/core/core/80-template.html [hunk 3]
```diff
@@ -33,6 +33,7 @@
                 <option value="handlebars">mustache</option>
                 <option value="html">HTML</option>
                 <option value="json">JSON</option>
+                <option value="yaml">YAML</option>
```
Comment (dceejay) [inline nodes/core/core/80-template.html]: know you love it - but can we add to bottom of list (well below CSS anyway ?

Comment (natcl) [inline nodes/core/core/80-template.html]: sure !

**Comment by natcl** [comment]
All changes made !

**Comment by natcl** [comment]
Test added !

**Comment by dceejay** [comment]
Thanks @natcl

**Pull request merged**